backend/src/components/utils.js

```diff
@@ -71,7 +71,63 @@ const utils = {
 
   // TODO: this is likely soon to be refactored out, as we will be pulling acronyms from the DB, not from access roles
   // Returns only app acronym based roles
-  filterAppAcronymRoles: roles => roles.filter(role => !role.match(/offline_access|uma_authorization|WEBADE_CFG_READ|WEBADE_CFG_READ_ALL/))
+  filterAppAcronymRoles: roles => roles.filter(role => !role.match(/offline_access|uma_authorization|WEBADE_CFG_READ|WEBADE_CFG_READ_ALL/)),
+
+  // From the big list of webade configs, return all APPLICATION preferences that match the search critera in the name
+  // that are not masked
+  filterForInsecurePrefs: (webadeConfigsList, searchCriteria) => {
```

Comment:
Would be good to convert this into JSDoc format.

Reply:
Done